e2e/tests/ui-driven/src/helpers/getPage.ts

```diff
@@ -1,8 +1,5 @@
-import { Browser, Page, Request } from "@playwright/test";
-import { createAuthenticatedSession } from "../globalHelpers";
-
-export const isGetUserRequest = (req: Request) =>
-  req.url().includes("/user/me");
```

Comment:
Renamed the file to be more specific to 'getPage' helpers, and moved this to another file